tests/unit/test_report_show_e2e.py

```
@@ -245,3 +245,92 @@ def capture_send(report_self):
+		time.sleep(0.05)
+		self._write_report('workflows', 1, [MEDIUM_VULN.copy()])
+		result, captured = self._invoke('last')
+		self.assertIsNone(result.exception, str(result.exception))
+		self.assertEqual(result.exit_code, 0)
+		vulns = captured.get('results', {}).get('vulnerability', [])
+		self.assertEqual(len(vulns), 1)
+		self.assertEqual(vulns[0]['name'], 'XSS')
+
+	def test_last_no_reports_exits_cleanly(self):
+		"""'last' with no reports exits cleanly (falls back to showing all)."""
+		result, captured = self._invoke('last')
```

Ruff (0.15.15)

[warning] Unpacked variable `captured` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)